devtools/common/src/main/resources/templates/basic-rest/kotlin/build.gradle-template.ftl

```diff
@@ -32,6 +32,11 @@ dependencies {
 group '${project_groupId}'
 version '${project_version}'
 
+test {
+    useJUnitPlatform()
+    exclude '**/Native*'
```

Comment:
Don't we need a way to run the native tests somehow?

Reply:
Yes we do (I'm currently looking into that). However if one doesn't exclude the native tests the test job fails completely.

I'll update the PR when I have a way to run the native tests as well.

Reply:
@gsmet when you say different directories you mean still under `src/test` or having a completely new source set like `src/it` (which seems to be the gradle way of doign things)?

Reply:
I remember seeing someone doing exactly that. Let me try to find it.

Reply:
I'm working on it. I have it half working, but let's see how the other half goes :)

Reply:
I can't find it. But the idea IMHO is too be in line with the Gradle way. We need to be sure we can extend one test from an IT though, not sure how the test classpaths are built.

Reply:
It looks like there is a general issue with the native tests and gradle... Haven't pinpointed it yet, I'll update when I do.

Reply:
The classpath thing I think I have figured out. The actual test execution is causing a lot problems

Reply:
I think I've come up with a proper solution, I'll update shortly

Reply:
I added a new commit that adds a source set. I tested on both Java and Kotlin projects and the tests were ran correctly.

Reply:
Actually it's not ready... Seems like something is missing...